src/js/components/Notification/stories/typescript/Status.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+
+import { Box, Grommet, Notification, Text } from 'grommet';
+
+import { deepMerge } from 'grommet/utils';
+import { grommet, ThemeType } from 'grommet/themes';
+import { DisabledOutline } from 'grommet-icons';
+
+// Type annotations can only be used in TypeScript files.
+// Remove ': ThemeType' if you are not using Typescript.
+const customTheme: ThemeType = deepMerge(grommet, {
+  global: {
+    font: {
+      family: 'Arial',
+    },
+  },
+  notification: {
+    container: {
+      pad: { horizontal: '200px', vertical: '100px' },
+      background: '#aaccff',
```

Comment:
Maybe add another Box prop such as `round`?

```suggestion
      background: '#aaccff',
      round: 'small',
```